bot/pagination.py

```diff
@@ -261,8 +261,8 @@ def event_check(reaction_: Reaction, user_: Member) -> bool:
 
                 await message.edit(embed=embed)
 
-        log.debug("Ending pagination and removing all reactions...")
-        await message.clear_reactions()
+        log.debug("Ending pagination and deleting the message")
+        await message.delete()
```

Comment:
When `asyncio.TimeoutError` happened ( No more reaction from user in set amount of time ) then the default behavior will be to keep the result but remove all the reactions, if you change it to delete, the result from the query will also be gone.

In this case, we want to keep the message and only clear all of its reactions.